src/components/Lpcontent/Lpcontent.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import './Lpcontent.css';
+import Topservicecard from "./Topservicecard";
+
+
+
+
+function  Lpcontent(props) {
+    
+        return(
+        <div>
+           
+            <Topservicecard  topservicecat="Beauty" />
+            <Topservicecard topservicecat="Tutor"/>
+            <Topservicecard topservicecat="Personal Trainer"/>
+            <Topservicecard topservicecat="Web developer"/>
+            
+            
+            
+            </div>
+            )    
```

Comment:
styling please stick to the same format:
```
function Example(props) {
   return (
       <React.Fragment>
             ...content
       </React.Fragment>
   )
}
```

Reply:
Noted the open and closing brackets should have the same distance to the left hand side.
E.g. the closing bracket of return should be vertical align with the word return